src/main/java/Hien.java

```diff
@@ -0,0 +1,249 @@
+import java.io.*;
+import java.nio.file.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Hien {
+    private ArrayList<Task> tasks;
+    private static final String DATA_FILE_PATH = "data/tasks.txt";
+    private static final DateTimeFormatter INPUT_DATE_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+    private static final DateTimeFormatter OUTPUT_DATE_FORMAT = DateTimeFormatter.ofPattern("MMM d yyyy, hh:mm a");
```

Comment:
I like that you used final values for these fields as accidentally changing them would cause big problems for your program. Good use of SCREAMING_SNAKE_CASE to name these constants.